org.openkj.OpenKJ.yml

```diff
@@ -0,0 +1,70 @@
+app-id: org.openkj.OpenKJ
+runtime: org.kde.Platform
+runtime-version: 5.15
+sdk: org.kde.Sdk
+command: openkj
+finish-args: 
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--device=dri"
+  - "--share=network"
+    #  - "--socket=wayland"
+  - "--socket=pulseaudio"
+  - "--filesystem=host"
+  - "--env=LADSPA_PATH=/app/lib/ladspa/"
+  - "--env=QT_QPA_PLATFORM=xcb"
+  - "--env=GST_PLUGIN_PATH=/app/lib/codecs/lib/gstreamer-1.0"
+  - "--env=PATH=/app/lib/totem-pl-parser/bin/:/app/lib/codecs/bin/:/app/bin:/usr/bin"
+modules:
+  - codecs/liba52.json
+  - codecs/libmpeg2.json
+  - codecs/gst-plugins-ugly.json
+  - codecs/libass.json
+  - codecs/ffmpeg.json
+  - codecs/gst-libav.json
+  - codecs/gst-plugins-bad.json
+  - name: rubberband 
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/breakfastquay/rubberband.git
+        branch: default
```

Comment:
```suggestion
```
branch + commit is doomed to fail as the HEAD commit will change. Just a commit.

Reply:
fixed